src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -828,18 +754,34 @@ Tuple TetMesh::TetMeshOperationExecutor::collapse_edge()
             auto tt = tt_accessor.index_access().vector_attribute(t_ear_1);
             auto tf = tf_accessor.index_access().vector_attribute(t_ear_1);
             auto te = te_accessor.index_access().vector_attribute(t_ear_1);
+            auto tv = tv_accessor.index_access().vector_attribute(t_ear_1);
 
             for (int k = 0; k < 4; k++) {
```

Comment:
`k` represents the local face id, right? I feel like it makes the code easier to understand if you clarify it. Maybe name that variable `loc_f_id` or something similar.